src/modules/translation/languages/en.json

```diff
@@ -611,16 +611,16 @@
   "land_assign_ens_page": {
     "title": "Assign {name}",
     "subtitle": "Assign a name to {land}",
-    "set_name_message": "In order to set this name to your parcel you first need to send 2 transactions.",
+    "set_name_message": "Before you can assign this name to your parcel, you need to submit 2 transactions: one that translates the human readable, alphanumeric name into an Ethereum address and one that creates a hash code linking that generated address to your LAND.",
```

Comment:
This is not correct. Let's use:

```diff
- [...] one that translates the human readable, alphanumeric name into an Ethereum address and one that creates a hash code linking that generated address to your LAND.

+ one that allows the ENS public resolver to translate your name, and one that resolves your name with the desired link.
```

Reply:
isn’t this too technical? should a user even care about understanding what does ENS stand for?

Reply:
instead of describing what the tx does I would rather explain _why_ are we doing it. Sure we set the Resolver, because the Resolver allows us to map an ENS to an Ehereum address or an IPFS hash, but we do it _because we want to be able to link your name to a url_ (by mapping an IPFS hash to it and then resolving a URL using https://eth.link, but the users don't care about that).

I think it's fine saying something like "The first transaction allows you to use your name as a link" and "the second transaction links your name to your land" or something like that

Reply:
I totally agreed with what you are saying. I was thinking it for advanced users, only focused on using the right words for the exact action.

Both examples you put above are ok for me.

Reply:
Thanks guys! done!